pkg/apis/mattermost/v1alpha1/clusterinstallation_types.go

```diff
@@ -65,7 +72,11 @@ type DatabaseType struct {
 	// Defines the storage size for the database. ie 50Gi
 	// +optional
 	// +kubebuilder:validation:Pattern=^([+-]?[0-9.]+)([eEinumkKMGTP]*[-+]?[0-9]*)$
-	DbStorageSize string `json:"dbStorageSize,omitempty"`
+	DatabaseStorageSize string `json:"databaseStorageSize,omitempty"`
```

Comment:
Updated to match the naming conventions of the other fields

Reply:
👍